docs/vars.md

```diff
@@ -0,0 +1,96 @@
+Configurable Parameters in Kargo
+================================
+
+#### Generic Ansible variables
+
+You can view facts gathered by Ansible automatically
+[here](http://docs.ansible.com/ansible/playbooks_variables.html#information-discovered-from-systems-facts).
+
+Some variables of note include:
+
+* *ansible_user*: user to connect to via SSH
+* *ansible_default_ipv4.address*: IP address Ansible automatically chooses.
+  Generated based on the output from the command ``ip -4 route get 8.8.8.8``
+
+#### Common vars that are used in Kargo
+
+* *calico_version* - Specify version of Calico to use
+* *calico_cni_version* - Specify version of Calico CNI plugin to use
```

Comment:
there is also calico_node and other plugins as well. Please refer for more details to the network plugins doc.